src/components/Fingerprint.vue

```diff
@@ -166,5 +168,21 @@ export default class Fingerprint extends ( mixins( Messages ) as VueConstructor<
 			@include fontSize( 23px );
 		}
 	}
+	.wikibase-termbox-fingerprint-list & {
+		@include media-breakpoint-up( md ) {
```

Comment:
Flex - hurray.
Now to the `lg` break point (1036px+) which is supposed to show 4 columns.

Reply:
done